WeakEvent/WeakEvent.csproj

```diff
@@ -20,6 +20,14 @@
         <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
+        <DefineConstants>$(DefineConstants);NET35</DefineConstants>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(TargetFramework)' == 'net35'">
+        <PackageReference Include="TaskParallelLibrary" Version="1.0.2856" />
```

Comment:
Why do you need this package reference? WeakEvent doesn't use the TPL at all...

Reply:
It's for a ConcurrentDictionary backport.

Reply:
Ah, I see. Is there no official port ? I'm wary of taking a dependency on this package, because I know nothing about it... the source code doesn't seem to be available, and the download link in the release notes is broken.